server/init_defaults.py

```diff
@@ -0,0 +1,68 @@
+from server.exts import db
+from models import Post, Author
+import json 
+import random
+from random import randint
+
+def make_authors():
```

Comment:
this is for the database, right?

Reply:
yep! I kept it in case anyone needs the functions to generate their own dummy data